packages/office-addin-debugging/src/commands.ts

```diff
@@ -39,20 +53,41 @@ export async function start(manifestPath: string, appType: string | undefined, c
         const packager: string | undefined = command.packager || process.env.npm_package_scripts_packager;
         const packagerHost: string | undefined = command.PackagerHost || process.env.npm_package_config_packager_host;
         const packagerPort: string | undefined = command.PackagerPort || process.env.npm_package_config_packager_port;
+        const sourceBundleUrlComponents = new devSettings.SourceBundleUrlComponents(
+            command.sourceBundleUrlHost, command.sourceBundleUrlPort,
+            command.sourceBundleUrlPath, command.sourceBundleUrlExtension);
+
+        if (appPlatformToDebug === undefined) {
+            throw new Error("Please specify the application platform to debug.");
```

Comment:
Please omit "application". `Please specify the platform to debug.` is sufficient.